userstorage/mount.py

```diff
@@ -82,8 +83,11 @@ def setup(self):
     # Helpers
 
     def _create_filesystem(self):
-        # TODO: Use -t xfs (requires xfsprogs package).
-        subprocess.check_call(["sudo", "mkfs", "-q", self._loop.path])
+        if self.fstype:
```

Comment:
could be simplified into
# mkfs default type
type = "ext2"
if self.fstype:
   type = self.fstype
 subprocess.check_call(["sudo mkfs", "-t", type, "-q", self._loop.path])

Reply:
+1

Reply:
I don't think ext2 is a good default, but using default sounds better than using base mkfs. I avoided -t because it failed on Fedora 28, but I think the issue was that -t was not the first argument. I'll improve this in the next version.